docs/configuration/memory.md

```diff
@@ -47,6 +47,17 @@ You can specify the memory index for redis using the following:
 MEMORY_INDEX=<WHATEVER>
 ```
 
+#### Using docker-compose
+In addition to the above changes in environment variables, change this one:
+```sh
+REDIS_HOST=redis-stack-server
```

Comment:
that would explain why you are naming the container. use the service name instead (redis), and docker will take care of the rest